src/plugins/vis_type_vega/public/vega_view/vega_base_view.js

```diff
@@ -297,6 +298,17 @@ export class VegaBaseView {
         this._addDestroyHandler(() => tthandler.hideTooltip());
       }
 
+      if (this._parser.visInteractions) {
+        this._parser.visInteractions.forEach((interaction) => {
+          if (interactionHandlersByAction[interaction.handlerName]) {
+            view.addEventListener(
+              interaction.event,
+              interactionHandlersByAction[interaction.handlerName]
+            );
```

Comment:
nit - prefer destructuring

```suggestion
        this._parser.visInteractions.forEach((interaction) => {
          const { handlerName, event } = interaction;
          if (interactionHandlersByAction[handlerName]) {
            view.addEventListener(
              event,
              interactionHandlersByAction[handlerName]
            );
```